libmscore/articulation.cpp

```diff
@@ -166,7 +167,6 @@ void Articulation::write(XmlWriter& xml) const
       for (const StyledProperty& spp : *styledProperties())
             writeProperty(xml, spp.pid);
       Element::writeProperties(xml);
-      writeProperty(xml, Pid::ARTICULATION_ANCHOR);
```

Comment:
This is now a styled property so it is written now as a part of the loop over `styledProperties()` above (at lines 166-167). This caused change in order of tags being written so I had to update mtest references for that purpose.